qtodotxt/ui/controllers/main_controller.py

```diff
@@ -381,7 +381,13 @@ def revert(self):
     def openFileByName(self, filename):
         logger.debug('MainController.openFileByName called with filename="{}"'.format(filename))
         self._fileObserver.clear()
-        self._file.load(filename)
+        if not self._file.load(filename):
+            currentfile = self._settings.value("last_open_file", "")
+            if currentfile == filename:
+                self._dialogs.showError(self.tr("Current file '{}' was deleted".format(filename)))
+            else:
+                self._dialogs.showError(self.tr("Error opening file: {}".format(filename)))
```

Comment:
yes it is not C, on different platforms work differently.....
 so moved processing to the next level